webrtc/src/peer_connection/certificate.rs

```diff
@@ -1,49 +1,45 @@
-use crate::dtls_transport::dtls_fingerprint::RTCDtlsFingerprint;
-use crate::error::{Error, Result};
-use crate::peer_connection::math_rand_alpha;
-use crate::stats::stats_collector::StatsCollector;
-use crate::stats::{CertificateStats, StatsReportType};
-
 use dtls::crypto::{CryptoPrivateKey, CryptoPrivateKeyKind};
-use rcgen::{CertificateParams, KeyPair, RcgenError};
+use rcgen::{CertificateParams, KeyPair};
 use ring::signature::{EcdsaKeyPair, Ed25519KeyPair, RsaKeyPair};
 use sha2::{Digest, Sha256};
+
 use std::ops::Add;
 use std::time::{Duration, SystemTime, UNIX_EPOCH};
 
-/// Certificate represents a x509Cert used to authenticate WebRTC communications.
-#[derive(Clone)]
-pub struct RTCCertificate {
-    pub(crate) certificate: dtls::crypto::Certificate,
-    pub(crate) stats_id: String,
+use crate::dtls_transport::dtls_fingerprint::RTCDtlsFingerprint;
+use crate::error::{Error, Result};
+use crate::peer_connection::math_rand_alpha;
+use crate::stats::stats_collector::StatsCollector;
+use crate::stats::{CertificateStats, StatsReportType};
 
-    pem: String,
-    expires: SystemTime,
+/// Certificate represents a X.509 certificate used to authenticate WebRTC communications.
+#[derive(Clone, Debug)]
+pub struct RTCCertificate {
+    /// DTLS certificate.
+    pub dtls_certificate: dtls::crypto::Certificate,
```

Comment:
The other option is to provide `to_bytes()` and `from_bytes()` methods, which will serialise/deserialise `RTCCertificate`. Let me know if you think this is better than making fields public.

Reply:
done ✅